crates/nu-command/src/conversions/into/datetime.rs

```diff
@@ -67,8 +82,11 @@ impl Command for IntoDatetime {
             (Type::String, Type::Date),
             (Type::List(Box::new(Type::String)), Type::List(Box::new(Type::Date))),
             (Type::table(), Type::table()),
+            // FIXME: https://github.com/nushell/nushell/issues/15485
+            // 'record -> any' was added as a temporary workaround to avoid type inference issues. The Any arm needs to be appear first.
+            (Type::record(), Type::Any),
             (Type::record(), Type::record()),
-            (Type::Nothing, Type::table()),
```

Comment:
We still need the `nothing -> table` for the usage with the `--list` flag to show the options for the format string.

I think this is a somewhat iffy way of showing documentation in Nushell but still widespread outside.